rest/views.py

```diff
@@ -89,8 +90,15 @@ class VocabularyList(ListCreateAPIView):
 
     def get_queryset(self):
         """Filter to vocabularies for a repository"""
-        return Repository.objects.get(
-            slug=self.kwargs['repo_slug']).vocabulary_set.order_by('id')
+        queryset = Vocabulary.objects.filter(
+            repository__slug=self.kwargs['repo_slug']
+        )
+        learning_resource_type = self.request.query_params.get(
+            'learning_resource_type', None)
+        if learning_resource_type is not None:
+            queryset = queryset.filter(
+                learning_resource_types__name=learning_resource_type)
```

Comment:
Wouldn't this be better by slug?  We don't have any types with weird names because they have to be valid XML, but semantically it would make more sense to do this by slug instead of name, or by type id.  Also, this kind of makes the querystring a bit ambiguous since it doesn't really say in the name what it is expecting.  Maybe `type_slug` or `resource_slug` or something.